python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -108,14 +117,14 @@ def repackage(self, phosim_amp_files, out_dir='.'):
 
         segments = {}
         for filename in phosim_amp_files:
-            phosim_channel = os.path.basename(filename).split('_')[6][1:]
+            phosim_channel = os.path.basename(filename).split("_")[6][1:]
             segments[phosim_channel] = fits.open(filename)[0]
 
         # Read raft name from the first filename
-        raft = os.path.basename(filename).split('_')[4]
-        ccdslot = os.path.basename(filename).split('_')[5]
-        sensorId = '%s_%s'%(raft, ccdslot)
-        print('Repackaging sensor %s '%(sensorId))
+        raft = os.path.basename(filename).split("_")[4]
+        ccdslot = os.path.basename(filename).split("_")[5]
+        sensorId = "%s_%s" % (raft, ccdslot)
+        print("Repackaging sensor %s " % (sensorId))
```

Comment:
You could consider to write the format word to improve the readability. Not only here, but also other places as well:
```
print(f"Repackaging sensor {sensorId}")
```